portal-api/api-tests/fixtures/mandatoryCriteria.js

```diff
@@ -21,63 +21,63 @@ const MANDATORY_CRITERIA = [
         ],
       },
       {
-        id: '2',
+        id: 2,
         title: 'Financial',
         items: [
           {
             id: 4,
-            copy: 'The Bank Customer (to include both the Supplier and any Parent Obligor) is an <a href="/assets/files/financial_difficulty_model_1.1.0.xlsx" class="govuk-link">Eligible Person spreadsheet</a>',
+            copy: 'Where the supplier is not a “Person Within Scope of Windsor Framework”, it is an eligible person OR',
+          },
+          {
+            id: 5,
+            copy: 'Where the supplier is a “Person Within Scope of Windsor Framework”, both it and its parent obligor (if any) is an eligible person.',
           },
         ],
       },
       {
-        id: '3',
+        id: 3,
         title: 'Credit',
         items: [
           {
-            id: 5,
+            id: 6,
             copy: 'The Bank Customer (to include both the Supplier and any UK Parent Obligor) has a one-year probability of default of less than 14.1%.',
           },
         ],
       },
       {
-        id: '4',
+        id: 4,
         title: 'Bank Facility Letter',
         items: [
           {
-            id: 6,
+            id: 7,
             copy: 'The Bank Facility Letter is governed by the laws of England and Wales, Scotland or Northern Ireland.',
           },
         ],
       },
       {
-        id: '5',
+        id: 5,
         title: 'Legal',
         items: [
           {
-            id: 7,
+            id: 8,
             copy: 'The Bank is the sole and beneficial owner of, and has legal title to, the Transaction.',
           },
           {
-            id: 8,
+            id: 9,
             copy: 'The Bank has not made a Disposal (other than a Permitted Disposal) or a Risk Transfer (other than a Permitted Risk Transfer) in relation to the Transaction.',
           },
           {
-            id: 9,
+            id: 10,
             copy: 'The Bank’s right, title and interest in relation to the Transaction is clear of any Security and Quasi-Security (other than Permitted Security) and is freely assignable without the need to obtain consent of any Obligor or any other person.',
           },
           {
-            id: 10,
+            id: 11,
             copy: 'The Bank is not restricted or prevented by any agreement with an Obligor from providing information and records relating to the Transaction.',
           },
         ],
       },
     ],
   },
-  {
-    version: 1,
-    criteria: [],
-  },
```

Comment:
`portal-api\api-tests\v1\mandatory-criteria\mandatory-criteria.api-test.js` is affected by this change - `const oldMandatoryCriteria = allMandatoryCriteria[1];` is now undefined

Reply:
Good point @MarRobSoftwire have added.